core/src/test/java/org/springframework/security/saml/spi/opensaml/OpenSamlImplementationTest.java

```diff
@@ -1,17 +1,18 @@
 package org.springframework.security.saml.spi.opensaml;
 
 import org.junit.jupiter.api.Test;
+import org.opensaml.saml.saml2.core.NameID;
 import org.springframework.core.io.ClassPathResource;
-import org.springframework.security.saml.saml2.authentication.AuthenticationRequest;
-import org.springframework.security.saml.saml2.authentication.Issuer;
-import org.springframework.security.saml.saml2.authentication.Scoping;
+import org.springframework.security.saml.saml2.authentication.*;
```

Comment:
Can you change your IDE to not do bulk imports `*`?

Reply:
Will do. Sorry. Will you merge locally or do you want a squashed commit in the feature branch?

Reply:
you can just add a commit to this PR. that's fine